spec/async/queue_spec.rb

```diff
@@ -80,6 +80,44 @@
 			expect(count).to be == repeats
 		end
 	end
+
+	context 'with a custom queue' do
+		class LIFOQueue
```

Comment:
Unfortunately RSpec does not introduce scope, so this is global.

Reply:
I just fixed this.

But I think the main problem with this PR was that it got stuck with naming the internal queue...

Reply:
Yes, will revisit this.